libs/ledger-live-common/src/wallet-api/converters.test.ts

```diff
@@ -0,0 +1,65 @@
+import BigNumber from "bignumber.js";
+import "../__tests__/test-helpers/setup";
+
+import { getWalletAPITransactionSignFlowInfos } from "./converters";
+import type { Transaction } from "../generated/types";
+import type { WalletAPITransaction } from "./types";
+
+const ethBridge = jest.fn();
+jest.mock("../generated/platformAdapter", () => {
+  return {
+    ethereum: {
+      getPlatformTransactionSignFlowInfos: function () {
+        return ethBridge();
+      },
+    },
+  };
+});
+
+describe("getPlatformTransactionSignFlowInfos", () => {
+  beforeEach(() => {
+    ethBridge.mockClear();
+  });
+
+  it("calls the bridge if the implementation exists", () => {
+    // Given
+    const tx: WalletAPITransaction = {
+      family: "ethereum",
+      amount: new BigNumber(100000),
+      recipient: "0xABCDEF",
+    };
+
+    // When
+    getWalletAPITransactionSignFlowInfos(tx);
+
+    // Then
+    expect(ethBridge).toBeCalledTimes(1);
+  });
+
+  it("uses its fallback if the bridge doesn't exist", () => {
+    // Given
+    const tx: WalletAPITransaction = {
+      family: "algorand",
+      mode: "send",
+      amount: new BigNumber(100000),
+      recipient: "0xABCDEF",
+    };
+
+    const expectedLiveTx: Partial<Transaction> = {
+      family: tx.family,
+      mode: "send",
```

Comment:
Unfortunately it seems we cannot mix `Transaction` & `WalletAPITransaction`
I think it's because we have `optOut` in `WalletAPITransaction` while `AlgorandOperationMode` doesn't have it
<img width="423" alt="Screenshot 2022-12-01 at 14 32 18" src="https://user-images.githubusercontent.com/3009753/205066130-d9b40e61-16b8-45bc-a9b5-d5a4188cff25.png">
<img width="579" alt="Screenshot 2022-12-01 at 14 33 23" src="https://user-images.githubusercontent.com/3009753/205066137-b6be1aac-fe40-49fb-b652-26f9566dae35.png">

Reply:
Ok fine 👍